compiler_tester/src/vm/eravm/system_contracts.rs

```diff
@@ -253,10 +253,15 @@ impl SystemContracts {
         yul_file_paths.push(Self::PATH_EVM_INTERPRETER.to_owned());
         let yul_optimizer_settings = era_compiler_llvm_context::OptimizerSettings::cycles();
         let yul_mode = YulMode::new(yul_optimizer_settings, true).into();
-        let yul_llvm_options = vec!["-eravm-jump-table-density-threshold", "10"]
-            .into_iter()
-            .map(|option| option.to_owned())
-            .collect();
+        let yul_llvm_options = vec![
+            "-eravm-jump-table-density-threshold",
+            "10",
+            "-tail-merge-only-bbs-without-succ=true",
```

Comment:
Are you sure it works?
In the first param the value is separated, whereas in others it is not.

Reply:
Yes, this works. For boolean options, we have to use '=', instead we will get the following error (at least for llc):
`llc: Too many positional arguments specified!`

Reply:
After we merge [this](https://github.com/matter-labs/era-compiler-llvm/pull/667), will update this PR and will separate values for integer options.